src/Simplex/Chat/Store.hs

```diff
@@ -1917,6 +1917,28 @@ getGroupMember db user@User {userId} groupId groupMemberId =
       |]
       (groupId, groupMemberId, userId)
 
+getGroupMemberById :: DB.Connection -> User -> GroupMemberId -> ExceptT StoreError IO GroupMember
```

Comment:
getGroupMember function above checks additionally by groupId - this seems redundant since groupMemberId uniquely identifies group member - in other places we either had or passed (Switch API for example required passing) extra data for it - for files INV it required extra (or more difficult) query I believe.. I decided to basically repeat getGroupMember but we can keep and use only this one

Reply:
> for files INV it required extra (or more difficult) query I believe

didn't get it. Why is it a problem to have groupId as well? The point of it was a sanity check.

Reply:
RcvFileTransfer is quite ad-hoc and doesn't have it, it does have group member id though - seemed easier to use that than to load group id into file transfers, or querying group_id from files just to get group member (query for which doesn't really need it).

Reply:
group member is required for its connection - to send direct message